wasp.go

```diff
@@ -17,7 +17,7 @@ import (
 )
 
 const (
-	DefaultCallTimeout           = 1 * time.Minute
+	DefaultCallTimeout           = 10 * time.Minute
```

Comment:
I don't think we should change that, common case for most services should be even lower, maybe `30s`, but `1m` is okay. Max value for most of the services including small batch requests.
Can you set your specific long timeout in your tests?

Reply:
yep. sounds good